### ISSUE TITLE
Centroid of path2D objects is different from the path3D object they were generated from 

### ISSUE THREAD

**Issue by sirlammpsalot** [mikedh/trimesh#1135]
Dear all,

I am having some trouble slicing a mesh.  Specifically, I am getting an incorrect centroid when I go to a path2D object via the section_multiplane() function.  I am new to trimesh so I am not sure if this is a bug proper, or if this is expected behaviour for some reason that I don't yet understand.  However, I would be grateful for any help fixing my problem. 

Background: I am generating meshes using pygmsh to use in a simulation, and to do a specific analysis I need to slice these to 2D slices and save to image files.  A key requirement of this is that the spatial relationship among meshes must be maintained.  By this I mean that the center of mass/centroid of the slice in 2D must be the same as it is in 3D.  The problem I am having is that trimesh keeps moving all the slices to the positive quadrant when I go to a path2D object.  Here is some minimal code to demonstrate the problem using a mesh of a cube in the neg x, neg y quadrant:

```python
                import numpy as np
                from pathlib import Path
                import pygmsh
                import trimesh
                from trimesh.interfaces.gmsh import load_gmsh
                from shapely.geometry import LineString
                
                # Load the mesh produce via pygmsh - mesh is a cube, so all z-normal 
                #  slices should have the same centroid
                meshPath = Path(cubeMeshPath)
                inputMesh = load_gmsh(meshPath.__str__())
                m = trimesh.Trimesh(vertices = inputMesh["vertices"], faces = inputMesh["faces"])
     
                # print the type and centroid of the mesh
                 print("Type: {}, centroid:{}".format(type(m), m.centroid))
                # output: Type: <class 'trimesh.base.Trimesh'>, centroid:[-18.01474355 -17.99994611   1.03254294]
                # the centroid is correct - cube is located in Quadrant III (negative x, negative y)
                
                # get all cross sections parallel to z-axis - using just one height for simplicity
                bounds      = list(m.bounds[0])
                slices = m.section_multiplane(plane_origin=bounds, 
                                                 plane_normal=[0,0,1], 
                                                 heights=[0.3])

                # Print the type and centroid of the slice
                print("Type: {}, centroid: {}".format(type(slices[0]), slices[0].centroid))
                # output: Type: <class 'trimesh.path.path.Path2D'>, centroid: [4.39411026 4.50029368]
                # the centroid has moved - the slice is located in the Quadrant I (pos, x, pos y)
```

First I input the mesh.  Then I slice it up using section_multiplate().  Here, the centroid of the slice has moved to [4.39411026 4.50029368] even though the centroid of the mesh is at [-18.01474355 -17.99994611   1.03254294].  Naively at least, I had expected the Path2D object to be a projection onto the X-Y plane and therefore have the same X and Y coordinates as the mesh.  Again, since this is a cube, the slices are squares and the centroids should have the same X and Y as the cube.  

So next I tried a path3D object using the section() function.  

```python
                # Now try a path3D object
                slice2 = m.section(plane_origin=m.bounds[0], plane_normal=[0,0,1])
                print("Type: {}, centroid: {}".format(type(slice2), slice2.centroid))
                # output: Type: <class 'trimesh.path.path.Path3D'>, centroid: [-18.04565 -18.        0.     ]
                # the centroid is correct for a Path3D object
```

Now the centroid of this one matches the centroid of the cube again.  So to see where things may be going wrong, I converted this path3D object to a path2D object using to_planar().

```python
                # but if I go back to a path2d object, the centroid is wrong again
                path2d = slice2.to_planar()
                print("Type: {}, centroid: {}".format(type(path2d[0]), path2d[0].centroid))
                # output: Type: <class 'trimesh.path.path.Path2D'>, centroid: [-0.0304221  0.       ]
                # the centroid is wrong again.  Moreover, it is also different from the Path2D 
                # centroid from section_multiplane()
```

And we're back to having a wrong centroid.  Curiously, this one is different from the centroid produced from section_multiplane().  My conclusion is that there is something I do not understand about Path2D objects such that my expectation for the output is wrong.  

So to summarise, I am trying to slice a cube but however I take the data to 2D, I get an incorrect centroid.  Am I doing something wrong, or is something broken here?  How can I ensure that my slices have the correct coordinates and do not get moved around.

Thanks very much!

**Comment by omaralvarez** [comment]
I have the same issue, basically vertex coordinates are not respected in 2D and cannot use the resulting path since I need it to be aligned and have the same centroid as the mesh in 3D. Same thing with the 3D path, it is perfect. 

I would love to know if there is a workaround for this.
